### DISCUSSION TITLE
有没有计划发布独立部署的server版本?

### DISCUSSION THREAD

**Discussion by plmcoder** [dream-num/univer#3588] [Ideas | 产品建议]
Hi, 这是一个非常棒的项目! 
就我个人而言, 非常期待能用于实际项目, 但是有一些困惑.
因为我原来的应用并不是基于容器的. 咱们这边加上docker 等容器环境反而让整个架构有点复杂,
不知咱们这边有没有计划发布传统的独立部署的server版本?  比如一个zip/exe啥的安装一下或者解压就能用的.

谢谢~

**Comment by plmcoder** [comment]
还有就是, 非常轻量的应用, 是否支持如sqllite, derby之类的内嵌数据库.

**Comment by zhang-ke-ke** [comment]
希望把导入导出剥离出来就行，毕竟这个是基础功能

**Comment by WuuuL** [comment]
我也希望

**Comment by scvzerng** [comment]
导入导出是阻碍从lucksheet 迁移到univer最大的阻力

**Comment by jikkai** [comment]
1. 我们已经提供了一键部署的脚本
2. Univer GO 会逐步提供相关能力

**Reply by WuuuL** [reply]
> 1. 我们已经提供了一键部署的脚本
> 2. Univer GO 会逐步提供相关能力

哇哦，那太好了，可是我没找到这个脚本在哪，能给个链接吗？
